test/addons/rbd-mirror/start

```diff
@@ -108,11 +108,11 @@ def configure_rbd_mirroring(cluster, peer_info):
 
     print("Creating VolumeReplicationClass")
     template = drenv.template("start-data/vrc-sample.yaml")
-    yaml = template.substitute(cluster=cluster)
+    yaml = template.substitute(cluster=cluster, scname="rook-ceph-block")
```

Comment:
scname is cryptic. How about `storage_class="rook-ceph-block"`?

Reply:
I agree, I would like to take this enhancement in the next PR. the important parts are completed now.